Source/WebCore/Modules/modern-media-controls/controls/media-controls.css

```diff
@@ -90,13 +90,17 @@
     display: none;
 }
 
-.media-controls > * {
+.media-controls > *,
+.media-controls::before,
+.media-controls::after {
```

Comment:
Do we also need to do this for `.media-controls-container > *`?

NIT: Can we `:is(::before, ::after)`?

Reply:
Only the controls themselves ought to fade, so I don't think we want siblings to have those styles. I'll try `:is(::before, ::after)`, you're always one step ahead on those new things!